### DISCUSSION TITLE
Need a business partner for an idea creation. Willing to give up 40% equity

### DISCUSSION THREAD

**Discussion by wsmithsyn** [community/community#146538] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I need someone to help me with my idea. I'm struggling with not alot of money to invest but I've heard from investors that I have a great idea. Sadly Investors are not willing to just invest in an idea. I need someone that can help me to build it and I'm willing to have a partnership or give up more equity. I have no friends or family that can invest. That's why I'm here.

**Comment by shinybrightstar** [comment]
Hi @wsmithsyn , we’re glad you’re here 😀

Thanks for sharing your project with the community. Unfortunately, we currently do not allow self-promotion, advertising  or job listings on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
